components/badges/badge.css

```diff
@@ -0,0 +1,59 @@
+/* General Badge Class */
+.badge {
+  align-items: center;
+  display: flex;
+  font-size: 0.5rem;
```

Comment:
use px unit when the value is less than 1rem

Reply:
I have used relative unit to make it responsive for all kind of users with different font sizes.